packages/core/useOTP/index.ts

```diff
@@ -0,0 +1,74 @@
+// eslint-disable-next-line no-restricted-imports
+import type { Ref } from 'vue'
+import { onBeforeUnmount, ref } from 'vue-demi'
```

Comment:
also i don't think vue-demi is currently a dependency of the repo. Build will probably fail